plugins/techdocs/src/reader/components/TechDocsReaderPageHeader/TechDocsReaderPageHeader.tsx

```diff
@@ -102,7 +103,7 @@ export const TechDocsReaderPageHeader = (
   const labels = (
     <>
       <HeaderLabel
-        label="Component"
+        label={entityMetadata?.kind || 'Entity'}
```

Comment:
For consistency with the other header labels, we probably want this to be capitalized. For example `component` should be `Component`. And edge case is `api` which should probably be `API`.

Do you have a suggestion for how we could solve this?

Reply:
Maybe we could define a map of how we want to rename each kind?

Reply:
For now, I've used the `capitalize` function.